draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -548,12 +565,21 @@ fragment as described in {{lorawan-schc-payload}}.
 * MAX_ACK_REQUESTS: 8
 
 As only 1 tile is used, its size can change for each downlink, and will be
-maximum available MTU.
+maximum available MTU for unicast. For multicast, it has to be the minimum MTU
+of all devices, RECOMMENDED value is 51.
 
-_Note_: The Fpending bit included in LoRaWAN protocol SHOULD NOT be used for
+_Note 1_: The Fpending bit included in LoRaWAN protocol SHOULD NOT be used for
 SCHC-over-LoRaWAN protocol. It might be set by the Network Server for other
 purposes but not SCHC needs.
 
+_Note 2_: IPv6 multicast addresses must be defined as per [RFC4291].  LoRaWAN
+multicast group definition in a network server and the relation between those
+groups and IPv6 groupID are out of scope of this document.
+
+_Note 3_: LoRa Alliance defined  {{lora-alliance-remote-multicast-set}} as
```

Comment:
I suggest you move this section up into LoRaWAN Architecture section (subsection Multicast)

Reply:
Agreed